tests/test_plots.py

```diff
@@ -16,6 +17,11 @@ def setup(self):
         self.psis = jnp.asarray(self.psis)
         self.rhos = jnp.asarray(self.rhos)
 
+    @pytest.fixture(autouse=True)
+    def teardown(self):
+        yield
```

Comment:
Why do you need to yield here ? A small comment would be useful

Reply:
LGTM otherwise

Reply:
It's because it's an end-of-test fixture, first the test runs and then this is called. This is how pytest proposes to write this kind of fixture.